src/domain/__tests__/invoice/index.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { Invoice } from "../../Invoice";
+import { invoiceMock } from "../../__mocks__/invoiceMock";
+describe("when i called the invoice...", () => {
+  it("id attribute are called", () => {
+    expect(invoiceMock.id).toStrictEqual("123");
+  });
+
+  it("type attribute are called", () => {
+    expect(invoiceMock.type).toBe("regular");
+  });
+
+  it("amount attribute are called", () => {
+    expect(invoiceMock.amount).toStrictEqual(20);
+  });
+
+  it("createdAt attribute are called", () => {
+    expect(invoiceMock.createdAt.toLocaleLowerCase()).toStrictEqual(
+      "02/11/2022"
+    );
+  });
+  it("updatedAt attribute are called", () => {
+    expect(invoiceMock.updatedAt.toLocaleLowerCase()).toStrictEqual(
+      "23/10/2022"
+    );
+  });
+  it("expired attribute are called", () => {
+    expect(invoiceMock.expired).toStrictEqual(false);
+  });
+  it("installments attribute are called", () => {
+    expect(invoiceMock.installments).toStrictEqual([]);
+  });
+  it("loanId attribute are called", () => {
+    expect(invoiceMock.loanId).toStrictEqual(
+      "LOA-15E96D5B-A2CD-446A-AFB8-46B71A2EC98E"
+    );
+  });
+
+  it("dueDate attribute are called", () => {
+    expect(invoiceMock.dueDate.toLocaleDateString()).toStrictEqual(
+      "10/17/2022"
+    );
+  });
+  it("custom methods are called", () => {
+    const getterMethodMock = jest
+      .spyOn(Invoice.prototype, "checkIfExpired")
+      .mockImplementation(() => false);
+
+    const InstaciaOfInvoice = new Invoice(
```

Comment:
@daniszcode uma sugestão: como boa prática é sempre aconselhado criar variáveis e funções com  a primeira letra minúscula, e classes com a primeira letra maiúscula.
Aqui poderia ficar assim:
```suggestion
    const instanceOfInvoice = new Invoice(
```
O que achas?
Aproveitei pra deixar tudo em inglês, pra não ficar misturado duas linguas no nome da variável.